alma/alma-8.x/common/disable_user_namespaces.sh

```diff
@@ -0,0 +1,4 @@
+#!/bin/bash
+
+echo "user.max_user_namespaces=0" > /etc/sysctl.d/userns.conf
+sysctl -p /etc/sysctl.d/userns.conf
```

Comment:
whats the motivation behind this patch? can you provide more details pls?

Reply:
> whats the motivation behind this patch? can you provide more details pls?

It's in the install.sh where this is called. I also specifically asked about whether this needed to be ported in chat today. All commits being copied are in the description as well.